code/drasil-docLang/lib/Drasil/Sections/Requirements.hs

```diff
@@ -111,6 +112,16 @@ mkMaintainableNFR refAddress percent lbl = cic refAddress (foldlSent [
   S "assuming the same development resources are available"
   ]) lbl nonFuncReqDom
 
+mkPortableNFR :: String -> [String] -> String -> ConceptInstance
+mkPortableNFR refAddress osList lbl
+  | null osList = cic refAddress (S "No operating systems specified.") lbl nonFuncReqDom
```

Comment:
We should probably error out on an empty list as it doesn't make sense to have a requirement with no operating system defined. That being said, @smiths would be a better source of reference for this.

Reply:
I personally agree with this!

Reply:
I agree as well.

Reply:
When you use `cic`, you are still creating a `ConceptInstance`. We want to use `error "text"` instead. Where "text" is your error message, in this case something like "No operating system(s) specified."